src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/swift/SwiftPermanentStorage.java

```diff
@@ -54,8 +69,9 @@ public SwiftPermanentStorage(Properties properties) throws PermanentStorageExcep
 
     private boolean checkProperties(Properties properties) {
         String[] propertiesSet = {
-                SapsPropertiesConstants.PERMANENT_STORAGE_TASKS_DIR,
-                SapsPropertiesConstants.SWIFT_CONTAINER_NAME
+            SapsPropertiesConstants.PERMANENT_STORAGE_TASKS_DIR,
+            SapsPropertiesConstants.SWIFT_CONTAINER_NAME,
+            SapsPropertiesConstants.SWIFT_OBJECT_STORE_KEY
```

Comment:
Ok. I will add the properties to test configuration files. 
I believe that the `SapsPropertiesConstants.SWIFT_OBJECT_STORE_KEY` property should be checked because it is necessary for the function of generating links, which is one of the requirements of the permanent storage interface. The archiver can pass any key as it does not use the function of generating links.

Reply:
It's good, make sense to me